__tests__/app.test.js

```diff
@@ -177,3 +177,62 @@ describe("GET /api/articles", () => {
 			});
 	});
 });
+describe("GET /api/articles/:article_id/comments", () => {
+	it("should send the correct status code (200) back when passed an article id of an article that contains comments", () => {
+		return request(app).get("/api/articles/1/comments").expect(200);
+	});
+	it("should send the correct status code back (400) when passed an article id that is invalid - the parameter should be an integer", () => {
+		return request(app).get("/api/articles/two/comments").expect(400);
+	});
+	it("should send the correct status code back (404) when passed an article id that is valid but doesnt exist", () => {
+		return request(app).get("/api/articles/90909/comments").expect(404);
+	});
```

Comment:
Our error tests will also need to check the error messages. e.g. "Not Found" or "Bad Request"